docs/data/ampli/migration.md

```diff
@@ -4,34 +4,305 @@ description: If you already have analytics implemented in your product you can e
 template: guide-last.html
 ---
 
-If you already have analytics implemented in your product and want to migrate, Amplitude makes it easy to switch from your existing Analytics provider.
+Ampli provides the benefits of type-safety, linting, and data validation to make sure that your analytics are accurate and trustworthy.
 
-## Considerations
+To migrate from Amplitude SDK to Ampli, you need to replace all calls using Amplitude SDK with calls using Ampli. However, the process can be done gradually and at your own pace.
 
-- The `context` parameter isn't supported in Ampli. This was originally based on per Source Templates which is a deprecated feature. To add more data on each event you can do so by creating a Middleware that sets `contextProp=true` on all events.
+## Progressive Migration
 
-    ```js
-    export const contextMiddleware:Middleware = (payload, next) => {
-      payload.event.event_properties = {
-        ...payload.event.event_properties,    contextProp: true,  };  next(payload);};
-    ```
+### Step 0 - Amplitude SDK Only
 
-- The `validation` parameter isn't supported in Ampli. Before, this flag enabled and disabled local schema validation in the SDK. The new Ampli SDKs no longer do local schema validation and instead rely on backend validation on Amplitude servers which you can view on the Event "sparklines" in the Events page in Data.
+This page assumes you have existing analytics instrumentation using the Amplitude SDK.
 
-## Migrate events 
+#### Code Example
 
-For example, if you're currently tracking events like:
+```typescript
+import * as amplitude from '@amplitude/analytics-browser';
 
-```ts
-analytics.track('Song Uploaded',{
-  device: 'iOS'
+amplitude.init(AMPLITUDE_API_KEY);
+amplitude.track('Song Played', { title: 'The best song in the world.' });
```

Comment:
Can we also add event level group example [here](https://pr-760.d19s7xzcva2mw3.amplifyapp.com/data/ampli/migration/#track), since there has a lot of related questions?